src/core/src/FabricPrepareRenderResources.cpp

```diff
@@ -168,7 +191,15 @@ void setFabricTextures(
         if (hasBaseColorTexture(mesh)) {
             const auto baseColorTextureImage = GltfUtil::getBaseColorTextureImage(model, primitive);
             assert(baseColorTextureImage);
-            baseColorTexture->setImage(*baseColorTextureImage);
+            baseColorTexture->setImage(*baseColorTextureImage, TransferFunction::SRGB);
+        }
+
+        const auto featureIdTextureCount = getFeatureIdTextureCount(mesh);
+        for (uint64_t j = 0; j < featureIdTextureCount; j++) {
+            const auto featureIdSetIndex = mesh.featureIdTextureSetIndexMapping[j];
+            const auto featureIdTextureImage = GltfUtil::getFeatureIdTextureImage(model, primitive, featureIdSetIndex);
+            assert(featureIdTextureImage);
+            mesh.featureIdTextures[j]->setImage(*featureIdTextureImage, TransferFunction::LINEAR);
```

Comment:
So the feature ID textures are compressed differently at tileset creation time because they're data not meant for direct visualization

Reply:
Yeah, the PNGs themselves are encoded with a linear transfer function (as the spec requires)